src/content/docs/dropins/product-details/pdp-containers.mdx

```diff
@@ -141,6 +128,17 @@ Triggered when the Add to Cart button is clicked. Receives an object with `sku`,
 onAddToCart: (values) => void;
 ```
 
+## zoomType
+
+This property can take one of two string values: `zoom` or `overlay`. It is used to enhance product image viewing by providing zoom or overlay functionality:
+
+- `zoom`: Provides a close-up view of product details.
```

Comment:
Provides a close-up view of the _product's main image from the carousel_. The clicked image is _not_ opened full screen. 
You can check this experience here: https://experience-qa.adobe.net/solutions/magento-datalake-storefront-pdp/assets/qa20240912080555/standalone/product.html?sku=VSW12. In this example the **closeButton** is also set to true.